src/components/modules/metadata/index.tsx

```
@@ -46,31 +49,114 @@ export function encryptEmail(email: string) {
-            value={encryptEmail(DOCUMENT_OWNER)}
-          />
-          <div data-testid="divider" className="w-full h-0.5 bg-neutral-950" />
-          <InformationRow label="Document Type" value={DOCUMENT_TYPE} />
-        </div>
+        {isFetching ? (
+          <div className="flex items-center justify-center w-full h-full">
+            <p className="text-neutral-950">Loading...</p>
+          </div>
+        ) : (
+          <>
+            {!!data ? (
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)